### ISSUE TITLE
ERROR: Saved models could not be loaded

### ISSUE THREAD

**Issue by awitney** [chklovski/CheckM2#6]
For information, just downloaded and installed using the "git clone" approach, but the testrun fails

```
$ checkm2 testrun
[07/12/2022 10:16:33 AM] INFO: Test run: Running quality prediction workflow on test genomes with 1 threads.
[07/12/2022 10:16:33 AM] INFO: Running checksum on test genomes.
[07/12/2022 10:16:33 AM] INFO: Checksum successful.
[07/12/2022 10:16:33 AM] INFO: Calling genes in 3 bins with 1 threads:
    Finished processing 3 of 3 (100.00%) bins.
[07/12/2022 10:17:04 AM] INFO: Calculating metadata for 3 bins with 1 threads:
    Finished processing 3 of 3 (100.00%) bin metadata.
[07/12/2022 10:17:05 AM] INFO: Annotating input genomes with DIAMOND using 1 threads
[07/12/2022 10:25:04 AM] INFO: Processing DIAMOND output
[07/12/2022 10:25:04 AM] INFO: Calculating completeness of pathways and modules.
[07/12/2022 10:25:09 AM] ERROR: Saved models could not be loaded.
```

This is on CentOS 7.9.2009

**Comment by awitney** [comment]
I tried both approaches, with the same error

thanks

**Comment by chklovski** [comment]
Hi,

Could you clone the latest commit and let me know the specific errors raised?

**Comment by awitney** [comment]
Hi, so the error probably helps with the problem:

`$ checkm2 testrun
[07/18/2022 09:22:05 AM] INFO: Test run: Running quality prediction workflow on test genomes with 1 threads.
[07/18/2022 09:22:05 AM] INFO: Running checksum on test genomes.
[07/18/2022 09:22:05 AM] INFO: Checksum successful.
[07/18/2022 09:22:05 AM] INFO: Calling genes in 3 bins with 1 threads:
    Finished processing 3 of 3 (100.00%) bins.
[07/18/2022 09:22:38 AM] INFO: Calculating metadata for 3 bins with 1 threads:
    Finished processing 3 of 3 (100.00%) bin metadata.
[07/18/2022 09:22:38 AM] INFO: Annotating input genomes with DIAMOND using 1 threads
[07/18/2022 09:30:44 AM] INFO: Processing DIAMOND output
[07/18/2022 09:30:45 AM] INFO: Calculating completeness of pathways and modules.
[07/18/2022 09:30:49 AM] ERROR: Saved models could not be loaded: cudaGetDevice() failed. Status: CUDA driver version is insufficient for CUDA runtime version
`

I am aware that there is an old'ish gpu on that machine. So I checked the latest commits on a second server (also CentOS 7.9.2009) which does not have a gpu , and testrun completed without error. So i guess an option to switch on/off the use of gpu/cuda would fix it?

thanks

**Comment by chklovski** [comment]
This *should* be now resolved in release [1.0.0](https://github.com/chklovski/CheckM2/releases/tag/1.0.0), please let me know if this issue resurfaces.
